Instance/application.make

```diff
@@ -223,10 +223,12 @@ MAIN_MODEL_FILE = $(strip $(subst .gmodel,,$(subst .gorm,,$(subst .nib,,$($(GNUS
 
 MAIN_MARKUP_FILE = $(strip $(subst .gsmarkup,,$($(GNUSTEP_INSTANCE)_MAIN_MARKUP_FILE)))
 
+MAIN_STORYBOARD_FILE = $(strip $(subst .gmodel,,$(subst .gorm,,$(subst .nib,,$($(GNUSTEP_INSTANCE)_MAIN_STORYBOARD_FILE)))))
```

Comment:
Why would we ever expect that a storyboard file ends with .gmodel or .gorm? The only real alternatives I see here are .nib and .xib.

Reply:
It should only end in .storyboard.  I copy/pasted this and then forgot to change it.   One of the dangers of coding at 3AM in the morning. ;)

Reply:
Should be resolved now.